README.md

```diff
@@ -1,14 +1,65 @@
 # Northcoders News API
 
-To run the project locally, you will need set up the enviroment variables. We have two databases, one for testing and one for development. We can do this by creating two .env files for your project: .env.test and .env.development. 
+The hosted version is avaliable at https://first-web-service-wn9h.onrender.com/api
 
-Within these files, assign the corresponding database names.
+The project involves building an API for the purpose of accessing application data programatically.  The final API should mimic a real world backend service,
+which provides the information to the front end architecture. 
 
-e.g.
+In order to run the program locally, read the following:
+
+Node-postgres will be used to facilitate interactions with the PSQL database. You will require:      
+Node.js: ">=6.0.0"
+Postgres: "^8.7.3"
+
+Next, you should clone (not fork) the following github repo https://github.com/northcoders/be-nc-news.
+
+Then create a new public GitHub repository where you will push your new code. Create a new branch for each task, pushing the branch to Github on completion of a task, then switching back to the main branch and pulling the changes down.
+
+In order to seed the local database, the enviroment variables need to be configurated. The are two seperate databases, one for testing and one for development. This can be done by creating two .env files for your project: .env.test and .env.development. 
 
+Within these files, assign the corresponding database names, with appropriate information relating to your PostgreSQL setup.
+
+e.g.
 env.test 
 PGDATABASE = test_db_name
 
```

Comment:
Make these instructions more obvious to a user. Imagine they have no idea what the right db name might be or which file to put what in. Tell them exactly what to write where